addon/-private/system/model/record-data-map.ts

```diff
@@ -0,0 +1,53 @@
+import { DEBUG } from '@glimmer/env';
+
+export const RECORD_DATA_KEY = `${Date.now()}-record-data`;
```

Comment:
Can you please write a comment of why/how we need the `RECORD_DATA_KEY` for

Reply:
👍